README.md

```diff
@@ -21,12 +21,18 @@ Write your secrets in secrets.yaml, see secrets.yaml.sample.
 The ./kops script supports a few subcommands and a number of arguments:
 
 ### Create a Cluster
-The create subcommand allows you to create a Kubernetes cluster within your AWS account.
+The create subcommand allows you to create a Kubernetes cluster within your AWS account. You must have AWS CLI configured and S3 bucket created.
+
+See [Setting AWS cluster](https://github.com/kubernetes/kops/blob/master/docs/aws.md) for more information.
```

Comment:
I'm not sure what you mean by "Setting AWS cluster" here to be honest, maybe we should use the original title of "Getting started on AWS"?